apps/meteor/server/startup/serverRunning.js

```diff
@@ -88,42 +88,42 @@ Meteor.startup(async () => {
 		showSuccessBox('SERVER RUNNING', msg);
 
 		// Deprecation
-		if (!skipMongoDbDeprecationCheck && semver.satisfies(semver.coerce(mongoVersion), '<6.0.0')) {
-			msg = [
-				`YOUR CURRENT MONGODB VERSION (${mongoVersion}) IS DEPRECATED.`,
-				'IT WILL NOT BE SUPPORTED ON ROCKET.CHAT VERSION 8.0.0 AND GREATER,',
-				'PLEASE UPGRADE MONGODB TO VERSION 6.0 OR GREATER',
-			].join('\n');
-			showWarningBox('DEPRECATION', msg);
-
-			const id = `mongodbDeprecation_${mongoVersion.replace(/[^0-9]/g, '_')}`;
-			const title = 'MongoDB_Deprecated';
-			const text = 'MongoDB_version_s_is_deprecated_please_upgrade_your_installation';
-			const link = 'https://go.rocket.chat/i/mongodb-deprecated';
-
-			if (!(await Users.bannerExistsById(id))) {
-				if (skipMongoDbDeprecationBanner || process.env.TEST_MODE) {
-					return;
-				}
-				sendMessagesToAdmins({
-					msgs: async ({ adminUser }) => [
-						{
-							msg: `*${i18n.t(title, adminUser.language)}*\n${i18n.t(text, { postProcess: 'sprintf', sprintf: [mongoVersion] }, adminUser.language)}\n${link}`,
-						},
-					],
-					banners: [
-						{
-							id,
-							priority: 100,
-							title,
-							text,
-							textArguments: [mongoVersion],
-							modifiers: ['danger'],
-							link,
-						},
-					],
-				});
-			}
-		}
+		// if (!skipMongoDbDeprecationCheck && semver.satisfies(semver.coerce(mongoVersion), '<7.0.0')) {
```

Comment:
we should keep the warning

Reply:
there is no version to be deprecated here so far, only 7.0.0